Networking/Networking/Model/Product/ProductCategory.swift

```diff
@@ -41,6 +41,14 @@ public struct ProductCategory: Decodable {
 
         self.init(categoryID: categoryID, siteID: siteID, parentID: parentID, name: name, slug: slug)
     }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
+        try container.encode(categoryID, forKey: .categoryID)
+        try container.encode(name, forKey: .name)
+        try container.encode(slug, forKey: .slug)
```

Comment:
You are right, in fact this PR not contains info for adding a new category. This probably will be changed in the next PR on which I'm working (which will allow us to add a new category and select a parent category). I'd say that in the next PR I'll remove `slug` because it seems generated from the backend if it's not present.